profile/README.md

```diff
@@ -0,0 +1,20 @@
+# HSE: Heterogeneous-Memory Storage Engine
+
+**HSE** is a fast embeddable key-value store designed for SSDs and
+persistent memory.
+HSE optimizes performance and endurance by orchestrating data
+placement across DRAM and multiple classes of solid-state storage.
+
+HSE is ideal for powering Databases, Software-Defined Storage (SDS),
+High-Performance Computing (HPC), Internet of Things (IoT), and Machine Learning (ML).
+
+## Getting Started
+
+It is easy to get started with HSE.
+
+* [Read the documentation to learn more about HSE](https://hse-project.github.io/)
```

Comment:
I don't really love these hyperlinks being the full bullet point.

1. Use `documentation` as the operative word
2. Use `HSE` as the operative word
3. Maybe substitute `HSE` for `corporate site` or something. Thoughts?

Reply:
I also don't like it, but I am not sure how you describe that site